bindings/go/Makefile.am

```diff
@@ -9,18 +9,26 @@ GOPATH=$(abs_top_builddir)/bindings/go/$(objdir)
 CGOCFLAGS=-I$(abs_top_builddir)/src -I$(top_srcdir)/src
 CGOLDFLAGS=-L$(abs_top_builddir)/src/ucp/$(objdir) -lucp -L$(abs_top_builddir)/src/ucs/$(objdir) -lucs
 UCX_SOPATH=$(abs_top_builddir)/src/ucp/$(objdir):$(abs_top_builddir)/src/ucs/$(objdir):$(abs_top_builddir)/src/ucm/$(objdir):$(abs_top_builddir)/src/uct/$(objdir)
+GOTMPDIR=$(GOPATH)/tmp
 
+export GOTMPDIR
 export GOPATH
 export CGO_CFLAGS=$(CGOCFLAGS)
 export CGO_LDFLAGS=$(CGOLDFLAGS)
 
-build:
+prepare:
+	$(AM_V_at)mkdir -p $(GOTMPDIR)
```

Comment:
is it used by go compiler?
can we make GOTMPDIR the target itself (otherwise, need `.PHONY: prepare`)

Reply:
Yes, if there's no GOTMPDIR it'll fail to compile. Go doesn't create it, only removes it's internal content inside it